### PR TITLE
Correcting some typos and spellings

### PR DESCRIPTION
**Pull request by liamfallon**

### CODE REVIEW

**Review by efiacor — APPROVED** [review]

File: krm-functions/gen-configmap-fn/go.mod [hunk 1]
```diff
@@ -44,6 +44,6 @@ require (
 	k8s.io/kube-openapi v0.0.0-20230525220651-2546d827e515 // indirect
 	k8s.io/utils v0.0.0-20230505201702-9f6742963106 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/kustomize/kyaml v0.14.2 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.14.2
```
Comment (johnbelamaric) [inline krm-functions/gen-configmap-fn/go.mod:47]: this change is weird, this is generated code.

**Review by johnbelamaric — COMMENTED** [review]
LGTM, except the one change in the go.sum, which I don't think is harmful but not sure why it's there?

File: krm-functions/gen-configmap-fn/go.mod (hunk 1 above)
Comment (liamfallon) [reply krm-functions/gen-configmap-fn/go.mod:47]: vscode's linter moaned about it, I'll revert this change...

**Comment by liamfallon** [comment]
Is this OK to go now?

**Review by lapentad — APPROVED** [review]

**Pull request merged**